lib/OpenQA/WebAPI/Plugin/ObsRsync.pm

```diff
@@ -8,6 +8,7 @@ use Mojo::File;
 use Mojo::URL;
 use Mojo::UserAgent;
 use POSIX 'strftime';
+use experimental 'signatures';
```

Comment:
Please enable signatures via `Mojo::Base` as we do it everywhere else, e.g. change the first `use`-line to be:

```
use Mojo::Base 'Mojolicious::Plugin', -signatures;
```